tests/unit/setup-context-test.js

```diff
@@ -79,6 +79,21 @@ module('setupContext', function(hooks) {
     test('it calls setContext with the provided context', function(assert) {
       assert.equal(getContext(), context);
     });
+
+    test('can be used for unit style testing', function(assert) {
+      context.owner.register(
+        'service:foo',
+        Service.extend({
+          someMethod() {
+            return 'hello thar!';
+          },
+        })
+      );
+
+      let subject = context.owner.lookup('service:Foo');
```

Comment:
I suppose the uppercase `Foo` was deliberate? could probably use a comment in that case

Reply:
Haha, nope, not intentional. Will fix.